server/app/routes/api/v1/teams/index.js

```diff
@@ -19,11 +19,42 @@ const models = db()
 const {
   user: User,
   team: Team,
-  organization: Organization
+  organization: Organization,
+  userTeam: UserTeam
 } = models;
 const Resource = Team;
 
 /** see documentation at https://www.npmjs.com/package/sequelize-handlers */
+
+router.use(function(req, res, next) {
+  req.options = {
+        where: {
+            id: req.params.id,
+        }, 
+        include: [
```

Comment:
You might also achieve a similar goal using [sequelize scope](http://docs.sequelizejs.com/en/latest/docs/scopes/). And in my opinion, the more logic you can shift to the model level, the better.